### PR TITLE
Dev android zingo mobile v0 0 1 5

### PR DESCRIPTION
**Pull request by juanky201271**
Changes made:

-    New Android ndk Version -> r24 (24.0.8215888)
-    New Gradle Version -> 6.9.1
-    Added new Android Arch -> x86_64
-    New Android API -> 30 (it's possible to go up to 31)
-    New temporal Logo, hopefully soon we have the new Logo.
-    New snapshots for testing.
-    download_memo = 'none' fixed.

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -133,12 +133,11 @@ android {
     }
 
     defaultConfig {
-        applicationId "com.zecwalletmobile"
+        applicationId 'com.ZingoMobile'
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
-        versionCode 01
+        versionCode 05
```
Comment (zancas) [inline android/app/build.gradle:139]: Why 05?

File: rust/android/buildphases/exportbuiltartifacts.sh [hunk 2]
```diff
@@ -4,9 +4,10 @@
 mkdir -p ../../android/app/src/main/jniLibs/arm64-v8a
 mkdir -p ../../android/app/src/main/jniLibs/armeabi-v7a
 mkdir -p ../../android/app/src/main/jniLibs/x86
+mkdir -p ../../android/app/src/main/jniLibs/x86_64
 
 # copy over .so files
+cp ../target/x86_64-linux-android/release/librust.so   ../../android/app/src/main/jniLibs/x86_64/
```
Comment (zancas) [inline rust/android/buildphases/exportbuiltartifacts.sh:10]: Ooopppsss...  this was missing?!  Nice find!

**Review by zancas — APPROVED** [review]
There are non-blocking comments.  I have not yet succeeded in manually testing that memo is set to none in my local device...

File: android/app/build.gradle (hunk 1 above)
Comment (juanky201271) [reply android/app/build.gradle:139]: Because I uploaded to google play 5 different versions already.

**Pull request merged**